src/test/resources/runners/regressionTests.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Regression tests" parallel="classes">
+    <listeners>
+        <listener class-name="reporting.TestListener"/>
+    </listeners>
+
+    <test name="Actions page tests">
+        <classes>
```

Comment:
The browser and localRun parameters are missed. I'm not able to run tests locally using this runner.